cmd/verify.go

```diff
@@ -26,7 +28,8 @@ func verifyFunc(cmd *cobra.Command, args []string) {
 
 	yamlFileName := args[0]
 
-	if err := validate.ValidateCSVManifest(yamlFileName); err != nil {
+	// TODO: return a pointer instead
```

Comment:
can you clarify this comment?

Reply:
TODOs are the things that are of low priority but need to be implemented in the future. For this point, I need to return struct pointers in `serialize.go` file (and other places).